lib/platform/github/gh-got-wrapper.js

```diff
@@ -2,12 +2,18 @@ const URL = require('url');
 const ghGot = require('gh-got');
 const delay = require('delay');
 const parseLinkHeader = require('parse-link-header');
+const endpoints = require('../../util/endpoints');
 
 let cache = {};
 
 async function get(path, opts, retries = 5) {
   /* eslint-disable no-param-reassign */
-  opts = Object.assign({}, opts);
+  const { host } = URL.parse(path);
+  const config = endpoints.find({ platform: 'github', host });
```

Comment:
Maybe use something else other than `config`? We already use it for many purposes